repos/os/include/net/tcp.h

```diff
@@ -47,13 +47,11 @@ class Net::Tcp_packet
 		uint32_t _seq_nr;
 		uint32_t _ack_nr;
 		unsigned _data_offset : 4;
-		unsigned _reserved    : 3;
```

Comment:
I doubt this works as intended as the class reflects the bit layout of a TCP packet and at this bit location are 3 reserved bits.

Reply:
Ah, you're right. Interestingly I did have this patch on my clang branch for a while and I do run various net related scenarios with it and so far haven't encountered any failures. Anyway, I'll drop the patch for now since I haven't found a nice way to silence unused variable warnings like this. [[maybe_unused]] and __attribute__((unused)) work with clang, but produce warnings under GCC. Void casting such members in constructor works, but this class for example does not have user defined constructor at all. Void casing in random methods works, but is rather ugly.